geoopt/manifolds/product.py

```diff
@@ -435,3 +437,96 @@ def origin(
             )
         tensor = self.pack_point(*points)
         return geoopt.ManifoldTensor(tensor, manifold=self)
+
+
+class StereographicProductManifold(ProductManifold):
+    """
+    Product Manifold for Stereographic manifolds.
+
+    Examples
+    --------
+    A Torus
+
+    >>> import geoopt
+    >>> sphere = geoopt.SphereProjection()
+    >>> torus = StereographicProductManifold((sphere, 2), (sphere, 2))
+    """
```

Comment:
This will apply the same scaling rules for new functions as they were in the Stereographic manifold. Basically, all you'll need
```suggestion
    """
    __scaling__ = Stereographic.__scaling__.copy()
```